### ISSUE TITLE
Command to request a full state overview of the vosekast

### ISSUE THREAD

**Issue by karlTGA** [gewv-tu-dresden/vosekast_control#18]
I need for the website a command, that give me a overview off the vosekast. I like to use that everytime the the site loads new and the web app know nothing. 

If the answer comes as a single message or all elements of the vosekast send serial his state doesnt matter for me.

**Comment by immunity-anomaly** [comment]
Can you use the print_diagnostics method of scale?

**Comment by karlTGA** [comment]
> Can you use the print_diagnostics method of scale?

No, I don't think, that the scale is the ride place for that. You can sepperate the mqtt publish function in the `state` setter and reuse that. So the vosecast get triggered and iterate through all devices and call that function `publish_state`. 

Example:
```    
@state.setter
def state(self, new_state):
        self._state = new_state
        self.logger.info(f"New state of pump {self.name} is: {new_state}")
        self.publish_state()

def publish_state(self):
        self.mqtt.publish_message(StatusMessage('pump', self.name, self.state))
```

**Comment by immunity-anomaly** [comment]
publish_state method does not exist on branch develop.

**Comment by immunity-anomaly** [comment]
please see commit https://github.com/gewv-tu-dresden/vosekast_control/commit/3dea82e2e6d8c21f07c15644ad81292d6b2af9cf for prototype of state_overview method

**Comment by karlTGA** [comment]
implemented with pull request #23
